internal/pgutil/parser_test.go

```diff
@@ -15,16 +15,16 @@
 // specific language governing permissions and limitations
 // under the License.
 
-package apmpq_test
+package pgutil_test
 
 import (
 	"os"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 
+	apmpq "go.elastic.co/apm/internal/pgutil"
```

Comment:
```suggestion
	"go.elastic.co/apm/internal/pgutil"
```

(and update references)